models/saved-pins.js

```diff
@@ -1,10 +1,15 @@
 const mongoose = require('mongoose');
+
 const Schema = mongoose.Schema;
 const savedPinsSchema = new Schema({
-    lat: {type: Number, min: -90, max: 90, required: true},
-    lng: {type: Number, min: -180, max: 180, required: true},
-    place_id: {type: String, required: true},
-    save_date: {type: Date, default: Date.now},
-})
+  lat: {
+    type: Number, min: -90, max: 90, required: true,
```

Comment:
Just curious - if there was a lint reason why this was broken into a new line vs `place_id: { ... },`? Trying to better understand our lint rules - this on seems a little inconsistent

Reply:
I just used vscode autofix linting. This may just be a mistake.